Core/PARStore.h

```diff
@@ -68,9 +68,11 @@ extern NSString *PARStoreDidSyncNotification;
 - (BOOL)writeBlobFromPath:(NSString *)sourcePath toPath:(NSString *)path error:(NSError **)error;
 - (nullable NSData *)blobDataAtPath:(NSString *)path error:(NSError **)error;
 - (BOOL)deleteBlobAtPath:(NSString *)path error:(NSError **)error;
+- (BOOL)deleteBlobAtPath:(NSString *)path usingTombstone: (BOOL)usingTombstone error:(NSError **)error;
```

Comment:
Just wondering if we should leak the word "tombstone" into the API. In theory, that is purely an implementation detail.

How about `deleteBlobAtPath:registeringDeletion:error:`

Reply:
Good plan.